### PR TITLE
WEBUI-247: add flag column 3.0.x

### PR DESCRIPTION
**Pull request by andreacornaglia**
backport of https://github.com/nuxeo/nuxeo-web-ui/pull/1153

### CODE REVIEW

**Review by richardsd — APPROVED** [review]

File: elements/nuxeo-results/nuxeo-document-content.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ import '../nuxeo-data-grid/nuxeo-document-grid-thumbnail.js';
 import './nuxeo-results.js';
 import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
 import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { FiltersBehavior } from '@nuxeo/nuxeo-ui-elements//nuxeo-filters-behavior.js';
```
Comment (mnixo) [inline elements/nuxeo-results/nuxeo-document-content.js:33]:
```suggestion
import { FiltersBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-filters-behavior.js';
```

**Review by mnixo — APPROVED** [review]

File: elements/nuxeo-results/nuxeo-document-content.js (hunk 1 above)
Comment (andreacornaglia) [reply elements/nuxeo-results/nuxeo-document-content.js]: thanks, removed the extra slash

**Review by mnixo — DISMISSED** [review]

File: i18n/messages.json [hunk 2]
```diff
@@ -361,6 +362,9 @@
   "documentContentView.datatable.header.subjects": "Subjects",
   "documentContentView.datatable.header.title": "Title",
   "documentContentView.datatable.header.type": "Type",
+  "documentContentView.datatable.flags.coldStorage": "Under cold storage",
+  "documentContentView.datatable.flags.legalHold": "Under legal hold",
+  "documentContentView.datatable.flags.retention": "Under retention",
```
Comment (mnixo) [inline i18n/messages.json:367]: Aren't we missing the `documentContentView.datatable.flags.favorite` key here?

Comment (andreacornaglia) [reply i18n/messages.json]: good catch! just added it!

**Review by mnixo — DISMISSED** [review]

**Review by semisse — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**